croutonjs/src/js/fetch-jsonp.js

```diff
@@ -0,0 +1,108 @@
+const defaultOptions = {
+	timeout: 30000,
```

Comment:
Note there is a timeout built-in to this library for jsonp URLs. The default was 5 seconds. I bumped it to 30. We could bump it to something obscenely high if we never want it to timeout.